### DISCUSSION TITLE
Gleam support for the Rouge Ruby syntax highlighting library

### DISCUSSION THREAD

**Discussion by aognio** [gleam-lang/gleam#3623] [Show and tell]
I just created this pull request:

https://github.com/rouge-ruby/rouge/pull/2070

Here's how it looks using the `rougify` command:

```
rougify highlight -l gleam /path/to/todomvc.gleam
```

![gleam-rouge-aognio](https://github.com/user-attachments/assets/b8b4f237-0d5f-4e72-8929-ea5b702df70e)

The same file can be found on Github here:

https://github.com/gleam-lang/example-todomvc/blob/main/src/todomvc.gleam

My goal creating this is supporting syntax highlighting for Gleam in learnxinyminutes.com where I created a cheatsheet:

https://learnxinyminutes.com/docs/gleam/

Any feedback and collaborations will be appreciated.

**Comment by lpil** [comment]
Really cool! Thank you!
